doc/sphinx/source/project/introduction.rst

```diff
@@ -4,189 +4,97 @@
+Due to the growing number of new deep learning accelerators, integrating kernel libraries with frameworks has become increasingly nontrivial. For each new deep learning accelerator, a custom kernel library integration must be implemented by a team of experts. This labor-intensive work is further complicated by the number of frameworks as illustrated in the diagram below using orange lines. 
 
+.. _figure-B:
+
+.. figure:: ../graphics/lorem-ipsum.png
+   :width: 555px
+   :alt: 
 
-nGraph solves this problem with nGraph bridges. A bridge takes a computational 
-graph and reconstructs it in the nGraph IR with a few primitive nGraph 
-operations. With the unified computational graph, kernel libraries no longer 
-need to be separately integrated to each deep learning framework. Instead, the 
-libraries only need to support nGraph primitive operations, and this approach 
-streamlines integration process for the backend.  
+Each framework must be manually integrated with each hardware-specific kernel library. Additionally, each integration is unique to the framework and its set of deep learning operators, view on memory layout, feature set, etc. Each connection that needs to be made increases the amount of work, resulting in a fragile setup that is costly to maintain.
 
+nGraph solves this problem with nGraph bridges. A bridge takes a computational graph from a supported framework and reconstructs it in the nGraph IR with a few primitive nGraph operations. With a unified computational graph, kernel libraries no longer need to be separately integrated into each deep learning framework. Instead, the ibraries only need to support nGraph primitive operations, and this approach streamlines the integration process for the backend.  
 
-Problem: Increasing number of kernels 
+
+Problem 3: Increasing the number of kernels 
 -------------------------------------
 
-Kernel libraries need to be integrated with multiple deep learning frameworks, 
-and this arduous task becomes even harder due to increased numbers of required 
-kernels for achieving optimal performance. The number of required kernels is 
-product of number of chip designs, data types, operations, and the cardinality 
-of each parameter for each operation. In the past, the number of required 
-kernels was limited, but as the AI research and industry rapidly develops, the 
-final product of required kernels is increasing exponentially. 
+As previously mentioned, Kernel libraries need to be integrated with multiple deep learning frameworks, and this already arduous task becomes even harder due to the greater number of a kernels for achieving optimal performance. The number of required kernels is based on the number of chip designs, data types, operations, and the cardinality of each parameter per operation. In the past, the number of required kernels was limited, but as AI research and industry continue to rapidly develop, the number of required kernels is exponentially increasing. 
```

Comment:
`kernel` should not be capitalized

Reply:
typo: `a kernels`

Reply:
I wonder if we could rework this a bit, maybe start with something like "Past deep learning research has been built on a small set of standard computational primitives (convolution, GEMM, etc.). But as research progresses and industrial applications of deep learning continue to develop, the number of required kernels is exponentially increasing." Then go into the stuff about number of chip designs, data types, cardinality, &c.